bool dms_status #0 ok, 1 emergency
